Update Element Timing API pages (#22846)
mdn/content: files/en-us/web/api/performanceelementtiming/id/index.md

## Examples

### Using `id`

> In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.

Reviewer: Nitpicking, I guess adding `elementtiming` attr doesn't start observation but acts as an identifier:

```suggestion
In this example, an {{HTMLElement("img")}} element is given an [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute so it can be identified during performance observation. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, which is the {{htmlattrxref("id")}} attribute of the image element.
```

Reviewer: Correct me if I'm wrong here :)

Author: hm, I think the `elementtiming` attribute alone does in fact signal to the browser to store performance data about the element even if there is no code yet that accesses this data using a `PerformanceObserver`. The `buffered` flag is used to access data from before observer creation.

> In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.

Reviewer 2: ```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.
```
